frontend/src/pages/agreements/details/Agreement.jsx

```diff
@@ -55,6 +55,8 @@ const Agreement = () => {
     }
     let changeRequests = useChangeRequestsForAgreement(agreement?.id);
 
+    const isDirectContract = agreement?.agreement_type == "CONTRACT" ? true : false;
+
```

Comment:
Should we put any kind of TODO or comment indicating this is not the long-term intent?  not sure if we have precedent for such